### PR TITLE
Fix arrow.get functionality when weekday is parsed alone

### PR DESCRIPTION
**Pull request by jadchaar**
## Pull Request Checklist

Thank you for taking the time to improve Arrow! Before submitting your pull request, please check all *appropriate* boxes:

- [x] 🧪 Added **tests** for changed code.
- [x] 🛠️ All tests **pass** when run locally (run `tox` or `make test` to find out!).
- [x] 🧹 All linting checks **pass** when run locally (run `tox -e lint` or `make lint` to find out!).
- [x] 📚 Updated **documentation** for changed code.
- [x] ⏩ Code is **up-to-date** with the `master` branch.

If you have *any* questions about your code changes or any of the points above, please submit your questions along with the pull request and we will try our best to help!

## Description of Changes

Closes: https://github.com/crsmithdev/arrow/issues/446

Replaces https://github.com/crsmithdev/arrow/pull/820 due to GitHub glitch.

### CODE REVIEW

**Comment by systemcatch** [comment]
Hey Jad, do we want to create regression test based on the bug in #446 ?

`assert arrow.get("Tuesday", "dddd").format("dddd") == 'Monday'`

**Comment by jadchaar** [comment]
@systemcatch I was thinking of creating one, but I did not know where to put it. Do you think it relates more to parsing than formatting?

**Comment by systemcatch** [comment]
I think where you've put them is fine, later this evening I can check then approve the PR.

File: tests/test_parser.py [hunk 1]
```diff
@@ -580,6 +580,96 @@ def test_parse_DDDD_only(self):
...
+            self.parser.parse("Thursday", "dddd"), "dddd"
+        ) == "Thursday"
+        arw_formatter.format(self.parser.parse("Fri", "ddd"), "ddd") == "Fri"
+        arw_formatter.format(self.parser.parse("Friday", "dddd"), "dddd") == "Friday"
+        arw_formatter.format(self.parser.parse("Sat", "ddd"), "ddd") == "Sat"
+        arw_formatter.format(
+            self.parser.parse("Saturday", "dddd"), "dddd"
+        ) == "Saturday"
+        arw_formatter.format(self.parser.parse("Sun", "ddd"), "ddd") == "Sun"
+        arw_formatter.format(self.parser.parse("Sunday", "dddd"), "dddd") == "Sunday"
```
Comment (systemcatch) [inline tests/test_parser.py:671]: Exhaustively tested! :wink:

**Pull request merged**